src/components/Lobby/Lobby.module.css

```diff
@@ -1,13 +1,43 @@
+.main {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+}
+
 .header {
     display: flex;
     justify-content: center;
     flex-direction: row-reverse;
 }
 
-.currentRooms {
+.container {
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+}
+
+.leftSide {
+    width: 30%;
+    display: flex;
+    justify-content: stretch;
+    flex-direction: column;
+    align-items: stretch;
+}
+
+.createRoom {
+display: flex;
+justify-content: center;
+flex-direction: row;
+flex-flow: row wrap;
+align-items: center;
+}
+
+.roomArea {
+    flex-grow: 1;
+    width: 60%;
```

Comment:
Instead of the 30% and 60% you can make the `.leftSide` flex grow 0 and the `.roomArea` flex grow 1